Get Started/typograpgy.html

```diff
@@ -0,0 +1,126 @@
+                <a class="side-bar-items active" href="../Get Started/typograpgy.html">Typography</a>
+                <a class="side-bar-items" href="../Get Started/conclusion.html">Conclusion</a>
+
+                <span class="side-bar-title">Components</span>
+                <div id="line">
+                    <hr class="hr-sideBar" />
+                </div>
+
+                <a class="side-bar-items" href="../Components/Avatar/avatar.html">Avatar</a>
+                <a class="side-bar-items" href="../Components/Alert/alert.html">Alert</a>
+                <a class="side-bar-items " href="../Components/Badge/badge.html">Badge</a>
+                <a class="side-bar-items " href="../Components/Button/button.html">Button</a>
+                <a class="side-bar-items " href="../Components/Cards/card.html">Card</a>
+                <a class="side-bar-items" href="../Components/Responsive-Image/image.html">Image</a>
+                <a class="side-bar-items " href="../Components/Input/input.html">Input</a>
+                <a class="side-bar-items" href="../Components/List/list.html">List</a>
+                <a class="side-bar-items" href="../Components/Toast/toast.html">Toast</a>
+                <a class="side-bar-items " href="../Components/Grid/grid.html">Grid</a>
+                <a class="side-bar-items " href="../Components/Modal/modal.html">Modal</a>
+                <a class="side-bar-items " href="../Components/Navigation/navigation.html">Navigation</a>
+            </div>
+
+            <div class="content">
+                <span class="title">Typography</span>
+                <p class="margin-bottom-1 p-content">Documentation and examples for Sparrow typography, including global
+                    settings, headings, body text, lists, and more.
+
+                </p>
+                <hr class="margin-bottom-1">
+                <!-- Headings -->
+                <span class="sub-title">Headings</span>
+                <p class="p-content">All HTML headings, &lt;h1> through &lt;h6>, are available.</p>
+
+                <div class="typo-container">
+                    <h1>h1. Technoracy Heading</h1>
+                    <h2>h2. Technoracy Heading</h2>
+                    <h3>h3. Technoracy Heading</h3>
+                    <h4>h4. Technoracy Heading</h4>
+                    <h5>h5. Technoracy Heading</h5>
+                    <h6>h6. Technoracy Heading</h6>
+                </div>
+
+
+                <iframe class="iframeHide" src="https://carbon.now.sh/embed/iWHctMQ5KhfHwLFc9PH4"
+                    style="width: 411px; height: 298px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <hr class="margin-bottom-1">
+
+
+
+            </div>
+        </div>
+    </div>
+    <script src="../../Assets/Prism/prism.js"></script>
+    <script src="../../main.js"></script>
```

Comment:
rest is good ! GTM

Reply:
Thanks  [swethaogeti](https://github.com/swethaogeti) for the review